src/axom/core/examples/CMakeLists.txt

```diff
@@ -61,3 +61,36 @@ if(AXOM_ENABLE_TESTS)
         NAME    core_utilities
         COMMAND core_utilities_ex ${CMAKE_CURRENT_SOURCE_DIR} )
 endif()
+
+axom_add_executable(
+    NAME        core_array_perf_ex
+    SOURCES     core_array_perf.cpp
+    OUTPUT_DIR  ${EXAMPLE_OUTPUT_DIRECTORY}
+    DEPENDS_ON  core
+    FOLDER      axom/core/examples )
+
+if(AXOM_ENABLE_TESTS)
+    # Run the marching cubes example on N ranks for each enabled policy
```

Comment:
You may want to change this comment, unless `core_array_perf` is the marching cubes example.

Reply:
Right.  Thanks.